docs/user_guide/install_server.md

```diff
@@ -25,12 +25,23 @@ When deploying with physical machines, the following conditions must be met.
 Required Software | Version Limit
 ---|---
 Git|1.9.1+
-JDK|1.7+
-Maven|3.0.3+
+Golang|1.12.x
 Nginx|0.8+
 
 ## Procedure - When Deploying with Docker
 
+### Get Supernode image
```

Comment:
I am wondering what is the correct spell. `Supernode` or `SuperNode` or `supernode`? `Dfdaemon` or `DFdaemon` or `dfdaemon`?